### ISSUE TITLE
[FE] Profile 2.0: Change alertbox heading level prop value to 2 to address the heading order issue

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#15908]
## Background

As part of re-running the axe scans for Profile 2.0 (#15707), @sanlouise found the following issue:

> Running the axe scans, we got one 'best practice' suggestion for when we display the profile-wide alert (which contains an h3) above the profile content, which contains an h2. 

![image](https://user-images.githubusercontent.com/14869324/98389786-c5343d00-2011-11eb-9a94-302c07ef80f9.png)

@jenstrickland advised that we fix this issue by making the following update:

> Thanks for checking on this. That alertbox heading should have the level prop value changed to 2 to address the heading order issue (alertbox developer documentation. Please let me know if I may help.

## Tasks

- [x] Change alertbox heading level prop value to 2

**Comment by Samara-Strauss** [comment]
Why did we close this ticket?

**Comment by sanlouise** [comment]
It is done and merged 😄

**Comment by Samara-Strauss** [comment]
Ah, thank you for letting me know!
